apps/website/src/pages/_app.tsx

```diff
@@ -6,10 +6,10 @@ import { SSRProvider } from "react-aria";
 import { trpc } from "@/utils/trpc";
 import "@/styles/globals.css";
 import Layout from "@/components/layout/Layout";
-import registerServiceWorker from "@/utils/sw";
+//import registerServiceWorker from "@/utils/sw";
 import { ConsentProvider } from "@/hooks/consent";
 
-registerServiceWorker("/RootServiceWorker.js");
+//registerServiceWorker("/RootServiceWorker.js");
```

Comment:
Do we want a quick bit of code to unregister it for existing folks?

Reply:
yeah probably best